src/overview/components/VisitAsListItem.css

```diff
@@ -1,11 +1,11 @@
 .root {
     position: relative;
     display: block;
-    width: 240px;
-    height: 120px;
+    width: 100%;
+    height: 10%;
```

Comment:
is this 10% of which element? if you don't know probably you want to use wh instead.